docs/rules/curly-bracket-spacing.md

```diff
@@ -0,0 +1,67 @@
+# Enforce spacing on the style of curly brackets. (curly-bracket-spacing)
```

Comment:
I think that `curly-bracket` has too wide meaning. There are `object-curly-spacing`, `template-curly-spacing`, and `block-spacing` in core. How about `mustache-curly-spacing` or `embedded-expression-curly-spacing`?